browser/app/js/actions.js

```diff
@@ -492,6 +492,16 @@ export const uploadFile = (file, xhr) => {
           message: 'Unauthorized request.'
         }))
       }
+      if (xhr.status == 500) {
+        setShowAbortModal(false)
```

Comment:
what is this change for? looks like unrelated. Would it be possible to send it later after this PR is integrated into master? . Perhaps you can send this separately as well? @dvstate

Reply:
The section of code added to actions.js is to allow the javascript alert dialog to display a descriptive and meaningful error message. Otherwise the user doesn't know specifically what the problem is. Quite a few users requested this ability during testing.

Reply:
Can you send this change as separate PR? @dvstate do not think this belongs to this work..